adapters/sharethrough/utils_test.go

```diff
@@ -472,6 +516,52 @@ func TestParseDomain(t *testing.T) {
 	}
 }
 
+/*
+	Copied over from: https://gist.github.com/alex-ant/aeaaf497055590dacba760af24839b8d
```

Comment:
Why not let the Go http libraries take care of this for you?

Reply:
Can you elaborate how I can do that? I tested having the ad server returning gzip responses and the response body in the `MakeBids` function was unusable until I decode it manually in the adapter.

Reply:
I'll take a closer look. According to Go's docs, it should be asking for gzip encoding by default and handling it transparently.